app/merchant/views.py

```diff
@@ -0,0 +1,9 @@
+from flask import render_template
+from . import merchant
+from flask.ext.login import login_required
+
+
+@merchant.route('/')
```

Comment:
Don't know. Though, I'd be cautious. As I said in Slack: What if there's a /profile route that vendors can view too?